app/assets/javascripts/organizations.js

```diff
@@ -10,9 +10,15 @@ $(document).ready(function() {
   if (('.org-address').length >= 1 ) {
     displayMap($('.org-address').text());
   }
+
+  //not set to display none so that the page doesn't jump up
+  $('.flash-notice').delay(1000).animate({ opacity: 0 });
```

Comment:
yeah, you're right - let's leave it like it is until we discover other cases where we'd want to dismiss it.